Dockerfile

```diff
@@ -4,7 +4,9 @@ ADD . /app/
 
 ENV TZ=Europe/Berlin \
     MENSTRUATION_TOKEN='TOKEN' \
-    MENSTRUATION_ENDPOINT=http://127.0.0.1:8000
+    MENSTRUATION_ENDPOINT=http://127.0.0.1:8000 \
+    MENSTRUATION_MODERATORS="" \
```

Comment:
isn't that the default?

Reply:
Yeah but this is okay and intended. In the container overview, everyone can see that there is an important setting that can be made here.